contributors/design-proposals/architecture/scope.md

```diff
@@ -0,0 +1,342 @@
+# Kubernetes scope
+
+Purpose of this doc: Clarify factors affecting decisions regarding
+what is and is not in scope for the Kubernetes project.
```

Comment:
It would be useful to explain here what the relevance of "in scope" vs "out of scope" is - i.e. what do those two labels mean, exactly, and therefore why should I read this document.  Later on there is sentence "Most decisions are regarding whether any part of the project should undertake efforts in a particular area. However, some decisions may sometimes be necessary for smaller scopes".  I think that may be what you're trying to convey, but it's not at all clear.  Also, there have previously been objections to this point of view, in the sense that open source contributors can undertake efforts anywhere they or their sponsors/employers choose to (although if their attempted contributions by necessity need to be reviewed others, they mayl run into problems). I think you'll need to reword this to make your point more clear.

Reply:
Ack

Reply:
This and other comments make me think there are even more "meta" issues to resolve, such as our goals for Kubernetes, the software, and for Kubernetes, the project, why the issue of scope is important for each, how the issue of scope applies to new proposed functionality, and how it applies to existing code.

I was going to make a separate pass to update the list of related documents below, but I'll take another look at a better way to organize this content, so that we don't just end up with 4-5+ overlapping documents with different levels of detail and currentness.